src/Fantomas/CodePrinter.fs

```diff
@@ -1395,6 +1397,9 @@ and genExpr astContext synExpr =
        genExpr astContext e -- "." -- ident +> genExpr astContext e1 -- " <- "  +> genExpr astContext e2
     | DotGet(e, (s,_)) ->
         let exprF = genExpr { astContext with IsInsideDotGet = true }
+        let a = 42
+        let ee = e
+        let ss = s
         addParenIfAutoNln e exprF -- (sprintf ".%s" s)
```

Comment:
I think this don't belong here :)

Reply:
Yep, my bad. I sometimes add these because of the debugger in Rider. It doesn't show the value of `e` and `s` when hitting `addParenIfAutoNln`.